codility_training/lessons.lesson02.Arrays.OddOccurrencesInArray/khw970421.js

```diff
@@ -0,0 +1,17 @@
+// you can write to stdout for debugging purposes, e.g.
+// console.log('this is a debug message');
+
+function solution(A) {
+    let size = A.length;                //A의 크기 저장
```

Comment:
```suggestion
    const size = A.length;                //A의 크기 저장
```
size의 경우 변수가 아니라 **상수**